src/lib/vault/getFormSubmission.ts

```diff
@@ -0,0 +1,40 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { dynamodbClient } from "@lib/awsServicesConnector";
+
+interface FormSubmission {
+  content: string;
+}
+
+export async function getFormSubmission(
+  formId: string,
+  submissionName: string,
+): Promise<FormSubmission | undefined> {
+  try {
+    const response = await dynamodbClient.send(
+      new GetCommand({
```

Comment:
I'd avoid the bandwidth consumption and let the database handle the filtering, unless there's a good reason to do it in the API.

Reply:
My bad! I should have checked the design doc more carefully. For some reason, in my head, this API path was going to be used for any response. I thought the `/new` path would just return a list of submission names that are new 😅.

@wmoussa-gc I am not sure I understand what you said. Do you mean have the `GetCommand` operation check the status of the submission that we are requesting?

Feels like something that should be done at the route level. I originally envisioned the `getFormSubmission` function to stay as simple as possible so that it can be used by other API features.

Reply:
> I originally envisioned the getFormSubmission function to stay as simple as possible so that it can be used by other API features.

That makes sense. I won't worry about fetching data that might not be needed (so feel free to ignore my previous comment). In this case, it's just one submission. For some reason, I thought we would be pulling more items.